CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 
 ### Changed
+- Enable modifying the BagBuilder limitations for hooks (#308)
```

Comment:
As this is a new feature, it should go into the "Added" section, not into the "Changed" section.